pkg/inventory/inventory-client_test.go

```diff
@@ -397,7 +397,7 @@ func TestGetClusterObjs(t *testing.T) {
 			if !tc.isError && err != nil {
 				t.Fatalf("unexpected error received: %s", err)
 			}
-			if !object.SetEquals(tc.clusterObjs, clusterObjs) {
+			if !object.IsSetEqual(tc.clusterObjs, clusterObjs) {
```

Comment:
Remove this change. If you want to advocate for a name change, please do it in another PR.